test/requests/reviews/reviewRequest-test.ts

```diff
@@ -0,0 +1,79 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../../src/app';
+import prisma from '../../../src/db/prisma.config';
+
+chai.use(chaiHttp);
+
+type ReviewPost = {
+  shelterId: number
+  cleanliness: number
+  safety: number
+  staff: number
+}
+
+type ShelterPost = {
+  name: string
+  streetAddress: string
+  state: string
+  zip: number
+  phoneNumber: string
+}
+
+const shelterData: ShelterPost = {
+  name: 'Queer Little Fellas',
+  streetAddress: '1234 Gay St.',
+  state: 'CO',
+  zip: 42069,
+  phoneNumber: '1234567890'
+}
+
+
+describe('Post Review', () => {
+  let shelter: any;
+
+  beforeEach(async () => {
+    shelter = await prisma.shelter.create({data: shelterData});
+  })
+
+  it('create a review with given attributes', async () => {
+    const reviewData: ReviewPost = {
+      shelterId: shelter.id,
+      cleanliness: 10,
+      safety: 10,
+      staff: 9.8 
+    }
+
+    chai
```

Comment:
All of these tests are giving false positives given the way you have handled async execution. For example I added 
```expect(res.body.data.id).to.equal(12312322) ``` And it passed. In the words of Brian, never trust a passing test! Make sure you get it to fail first.